scripts/kind-e2e/cluster2-config.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+kubeadmConfigPatches:
+- |
+  apiVersion: kubeadm.k8s.io/v1beta1
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  networking:
+    podSubnet: 10.245.0.0/16
+    serviceSubnet: 100.95.0.0/16
+    dnsDomain: cluster2.local
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
```

Comment:
duplicate of line 15 - is this intended? ...

Reply:
Yes, we want two worker nodes.